sdk/keyvault/azkeys/example_test.go

```diff
@@ -102,18 +102,19 @@ func ExampleClient_UpdateKeyProperties() {
 		panic(err)
 	}
 
-	resp, err := client.UpdateKeyProperties(context.TODO(), "key-to-update", &azkeys.UpdateKeyPropertiesOptions{
-		Tags: map[string]string{
-			"Tag1": "val1",
-		},
-		Properties: &azkeys.Properties{
-			RecoveryLevel: azkeys.DeletionRecoveryLevelCustomizedRecoverablePurgeable.ToPtr(),
-		},
-	})
+	resp, err := client.GetKey(context.TODO(), "key-to-update", nil)
+	if err != nil {
+		panic(err)
+	}
+
+	resp.Key.Properties.Tags = map[string]string{"Tag1": "val1"}
+	resp.Key.Properties.Enabled = to.Ptr(true)
```

Comment:
Is the default Disabled? Or is this likely already true?

Reply:
It's likely already true, but a sample that we provide in other languages